__tests__/infoRequestHandler-spec.ts

```diff
@@ -1,75 +1,101 @@
-import * as express from 'express'
-import * as nock from 'nock'
```

Comment:
nock doesn't support node fetch.

https://stackoverflow.com/questions/74656122/how-do-i-mock-node-js-fetch-http-requests-responses-in-node-18

I tried using the more sophisticated mocking built into undici but it wasn't working so I just used simple mocking